gravitee-apim-rest-api/gravitee-apim-rest-api-automation/gravitee-apim-rest-api-automation-rest/src/main/java/io/gravitee/apim/rest/api/automation/resource/SharedPolicyGroupsResource.java

```diff
@@ -58,7 +60,7 @@ public class SharedPolicyGroupsResource extends AbstractResource {
     @PUT
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
-    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_SHARED_POLICY_GROUP, acls = { RolePermissionAction.CREATE }) })
+    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_SHARED_POLICY_GROUP, acls = { CREATE, UPDATE }) })
```

Comment:
If we go with checks in the controller, maybe we should just remove filters and go the same way every where ? I think it will avoid our future selves getting messed up with what to use and where.